server/handlers/hosts.go

```diff
@@ -0,0 +1,101 @@
+package handlers
+
+import (
+	"github.com/gofrs/uuid"
+	"github.com/layer5io/meshkit/database"
+	"gorm.io/gorm"
+)
+
+type MesheryHostsPersistor struct {
+	DB *database.Handler
+}
+
+type MesheryHosts struct {
+	ID       uuid.UUID
+	Hostname string
+	Port     int
+}
+type MesheryHostsDisplay struct {
+	ID       uuid.UUID           `json:"id"`
+	Hostname string              `json:"hostname"`
+	Port     int                 `json:"port"`
+	Summary  hostIndivitualCount `json:"summary"`
+}
+type MesheryHostsContextPage struct {
+	Page        int                   `json:"page"`
+	PageSize    int                   `json:"page_size"`
+	Count       int64                 `json:"total_count"`
+	Registrants []MesheryHostsDisplay `json:"registrants"`
+}
+type hostIndivitualCount struct {
+	Count_model        int64
+	Count_component    int64
+	Count_relationship int64
+}
+
+func (h Handler) GetMesheryHosts(limit int, page int, search string) ([]MesheryHostsDisplay, int64, error) {
+	db := h.dbHandler.DB
+	var hosts []*MesheryHosts
+	var hostsCount int64
+	if limit == -1 {
+		if err := db.Table("hosts").Select("*").Count(&hostsCount).Error; err != nil {
+			return nil, 0, err
+		}
+		limit = int(hostsCount)
+	}
+	if err := db.Table("hosts").
```

Comment:
This is good change but little off (inconsistent) from how we are currently  handling meshmodels in DB.
Please consider moving your gorm handlers to [MeshKit](https://github.com/meshery/meshkit) (under meshmodel [pkg](https://github.com/meshery/meshkit/tree/master/models/meshmodel)) and instead use [registryManger](https://github.com/meshery/meshery/blob/master/server/cmd/main.go#L155)